airflow/api/common/mark_tasks.py

```diff
@@ -107,11 +107,11 @@ def set_state(
     :param tasks: the iterable of tasks or (task, map_index) tuples from which to work.
         ``task.dag`` needs to be set
     :param run_id: the run_id of the dagrun to start looking from
-    :param execution_date: the execution date from which to start looking (deprecated)
+    :param logical_date: the logical date from which to start looking (deprecated)
```

Comment:
Should we deprecate it in Airflow 3? Maybe something to do in the future prs

Reply:
There’s a later PR to remove this. I believe this is already done in the unified PR, just not included here to simplify review.